docs/how-tos/async.rst

```diff
@@ -1,5 +1,5 @@
 ==============================
-Run Hamilton in a Microservice
+Online transformations
```

Comment:
I think micro-service is better here than online transformation. The main point to take away is that you can run Hamilton in a web-service.